cdci_data_analysis/flask_app/dispatcher_query.py

```diff
@@ -518,17 +518,16 @@ def set_temp_dir(self, session_id, job_id=None, verbose=False):
             print('SETTEMP  ---->', session_id,
                   type(session_id), job_id, type(job_id))
 
-        td = 'temp'
+        suffix = ""
 
         if session_id is not None:
-            td += '_sid_' + session_id
+            suffix += '_sid_' + session_id
 
         if job_id is not None:
-            td += '_jid_'+job_id
+            suffix += '_jid_'+job_id
 
-        td = FilePath(file_dir=td)
-        td.mkdir()
-        self.temp_dir = td.path
+        td = tempfile.mkdtemp(suffix=suffix)
```

Comment:
yeah, since this creates unique temp dir, it will not be possible to find it in the test without special effort.